vllm/distributed/communication_op.py

```diff
@@ -30,3 +30,30 @@ def broadcast_tensor_dict(tensor_dict: Optional[Dict[Any, Union[torch.Tensor,
     if not torch.distributed.is_initialized():
         return tensor_dict
     return get_tp_group().broadcast_tensor_dict(tensor_dict, src)
+
+
+def send_tensor_dict(tensors: Dict[str, torch.Tensor],
```

Comment:
I think we should not expose it here. The `broadcast_tensor_dict` in the module level is a tech debt. All future calls of a communication op needs to know it's scope, either WORLD, or TP, or PP.

Let's remove the module level function.

Reply:
It makes sense, I've made the change